chart/templates/cron.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "aws-ecr-creds.fullname" . }}-cronjob
+  namespace: {{ include "aws-ecr-creds.namespace" . | quote }}
+  labels:
+{{ include "aws-ecr-creds.labels" . | indent 4 }}
+spec:
+  schedule: {{ .Values.cronSchedule | quote }}
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 2  
+  jobTemplate:
+    spec:
+      backoffLimit: 4
+      template:
+        metadata:
+          labels:
+{{ include "aws-ecr-creds.podLabels" . | indent 12 }}
+        spec:
+          serviceAccountName: {{ include "aws-ecr-creds.serviceAccountName" . }}
+          terminationGracePeriodSeconds: 0
+          restartPolicy: Never
+          containers:
+          - name: kubectl
+            imagePullPolicy: IfNotPresent
+            image: {{ .Values.image }}:{{ .Values.imageTag }}
+            env:
+              - name: AWS_ACCESS_KEY_ID
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.secretName }}-keys-secret
+                    key: AWS_ACCESS_KEY_ID
+              - name: AWS_SECRET_ACCESS_KEY
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.secretName }}-keys-secret
+                    key: AWS_SECRET_ACCESS_KEY
+            command:
+            - "bash"
+            - "-c"
+            - |
+              AWS_ACCOUNT={{ .Values.aws.account }}
+              AWS_REGION={{ .Values.aws.region }}
+              DOCKER_REGISTRY_SERVER=https://${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
+              DOCKER_USER=AWS
+              DOCKER_PASSWORD=`aws ecr get-login --region ${AWS_REGION} --registry-ids ${AWS_ACCOUNT} | cut -d' ' -f6`
```

Comment:
could add `set -e` to the script to ensure that the script fails here if the aws get-login command fails. I don't remember if kubernetes has that option set by default or not in the command field